kits/api-breaking-change-detector/samples/test_flow.py

```
@@ -0,0 +1,76 @@
+    }
+}
+
+headers = {
+    "Authorization": f"Bearer {BEARER_TOKEN}",
+    "Content-Type": "application/json",
+    "x-project-id": PROJECT_ID
+}
+
+# 4. Send Request
+print("Triggering Lamatic Workflow...")
+response = requests.post(API_URL, json=payload, headers=headers)
```

Ruff (0.16.1)

[error] Probable use of `requests` call without timeout

(S113)